### DISCUSSION TITLE
add instructions

### DISCUSSION THREAD

**Discussion by iccor56** [cardillan/mindcode#31] [Ideas]
i would suggest you add a link to mindustry functions in your front page (web page or executable)

links such as 
https://www.reddit.com/r/Mindustry/comments/ic9wrm/logic_in_60/
https://steamcommunity.com/sharedfiles/filedetails/?id=2268059244
would explain what the individual functions do and give you a hub for information

https://marketplace.visualstudio.com/items?itemName=Antyos.vscode-mlog is handy too

**Comment by francois** [comment]
Yup, very good points! I linked to these resources, as well as another one I know of, in 147ea5f16ba5f2d1d5484e559c8170664c8b0115.
